### ISSUE TITLE
2.5.9 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2559]
- #2489 
- #2498
- #2511
- #2545
- #2561

## Pick list
```
git cherry-pick 277e8020a1f228a96bd88ad29dd8f987f57f236f
git cherry-pick a2673b2a2278526079e11b5fa3b5190c9c1103b8
git cherry-pick a136c208bbea66071474c95e8db02bacd31d4503
git cherry-pick f513e2b0ed10e4aeca15fb2ef944e7ca39513bbf
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.5.8..HEAD --pretty='format:- %s'`
```
- Closes #2543 Retire theme-debug lando/ddev tooling. (#2545)
- Closes #2502: Update Analytics Dashboard link in toolbar. (#2511)
- Closes #2497 Reorder people row weight input boxes don't show values (#2498)
- Closes #2486: Remove tooling for Drupal console 😢. (#2489)
- Update composer branch alias in preparation for 2.7.0 pre-release.
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.5.9
